tests/e2e/containers/config.go

```diff
@@ -24,7 +24,7 @@ const (
 	// It should be uploaded to Docker Hub. OSMOSIS_E2E_SKIP_UPGRADE should be unset
 	// for this functionality to be used.
 	previousVersionOsmoRepository = "osmolabs/osmosis-dev"
-	previousVersionOsmoTag        = "v14.x-de392d2b-1674667154"
+	previousVersionOsmoTag        = "v14.x-6b742c84-1675172347"
```

Comment:
How were you able to push the update? Is it correctly auto-pushing to Docker hub now?

Reply:
we had https://github.com/osmosis-labs/osmosis/pull/3503 this workflow to push osmosis-dev images